src/main/java/io/lumigo/core/utils/AwsUtils.java

```diff
@@ -243,6 +243,24 @@ public static synchronized Span.READINESS getFunctionReadiness() {
         }
     }
 
+    public static int getJavaVersion() {
+        try {
+            String version = System.getProperty("java.version");
+            if (version.startsWith("1.")) {
```

Comment:
isn't it safer to just work with closed lists of versions in order to check if it's before 11? the list of before 11 won't increase...

Reply:
alternatively, there has ato a beeter way to parse. maybe split on .